src/templates/expressions/fields.ts

```diff
@@ -121,6 +142,7 @@ export class SubTableField implements TsExpression {
             ${this.fieldGroup.tsExpression()}
         }
     }[];
+    disabled?: boolean;
```

Comment:
Is it really needed?

I test with 

```javascript
(() => {
    kintone.events.on("app.record.edit.show", (ev) => {
    	ev.record.Table.disabled = true
    })
})();
```

But SubTable field is not disabled. It is misleading...